docker/puppetserver-standalone/Dockerfile

```diff
@@ -18,6 +31,29 @@ RUN apt-get update && \
     /usr/local/bin/lein && \
     gem install --no-doc bundler fpm
 
+RUN wget http://nightlies.puppet.com/apt/puppet6-nightly-release-"$UBUNTU_CODENAME".deb && \
+    dpkg -i puppet6-nightly-release-"$UBUNTU_CODENAME".deb && \
+    rm puppet6-nightly-release-"$UBUNTU_CODENAME".deb && \
+    apt-get update && \
+    apt-get install --no-install-recommends puppet-agent && \
+    git clone --single-branch --branch native_fix https://github.com/underscorgan/rake-compiler && \
```

Comment:
Do we want a note about why we're using a fork / should we move this over to `puppetlabs` org?

Reply:
I have a note down a couple of lines. I got a comment on my PR yesterday so I'm holding out hope that we'll be able to not use a fork, but if it lingers too long can move it to the puppetlabs org

Reply:
Ah sorry, totally missed that.

See also https://github.com/rake-compiler/rake-compiler/pull/160

Reply:
So the latest comment at https://github.com/rake-compiler/rake-compiler/pull/160#issuecomment-560029680 says to just `gem install --no-ri --no-rdoc rugged` like you would normally... and then copy the generated gem out of `GEM_HOME` from the build container.

That way, no `rake-compiler` is ever needed.